packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -289,7 +289,9 @@ void main() {
     expect(controller.text, ' blah2blah1');
     expect(controller.selection, const TextSelection(baseOffset: 0, extentOffset: 0));
     expect(find.byType(CupertinoButton), findsNothing);
-  }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.macOS }), skip: kIsWeb);
+  },
+  variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.macOS }),
+  skip: kIsWeb); // [intended] the web handles this on its own.
```

Comment:
super nit: do we usually put the closing parenthesis on a separate line? In case someone needs to specify a new parameter after the `skip` argument in the future they don't have to mess with the comment and indentation.

Reply:
Yeah, I just updated this. I still don't know exactly how to indent these, as we use two opening symbols on the testWidgets line and if we want to close them on separate lines they wind up looking weird.